src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/IcraEligibilityViewWorkExperienceReference.vue

```diff
@@ -2,19 +2,100 @@
   <Loading v-if="loadingStore.isLoading('icra_status_get')" />
   <v-container v-else>
     <Breadcrumb />
-    <h1>Hello world</h1>
-    {{ icraEligibilityId }}
-    {{ referenceId }}
+
+    <h1 class="mb-6">Employment experience reference</h1>
+
+    <p class="mb-8">
+      We sent an email to the following person to request a reference.
+    </p>
+
+    <v-row class="mb-12">
+      <v-col cols="12" md="6">
+        <div class="detail-block">
+          <strong class="detail-label">Name</strong>
+          <div class="detail-value">
+            {{ fullName || "Not available" }}
+          </div>
+        </div>
+
+        <div class="detail-block">
+          <strong class="detail-label">Phone number</strong>
+          <div class="detail-value">
+            {{ phoneNumber || "Not provided" }}
+          </div>
+        </div>
+
+        <div class="detail-block">
+          <strong class="detail-label">Email</strong>
+          <div class="detail-value">
+            {{ emailAddress || "Not available" }}
+          </div>
+        </div>
+      </v-col>
+    </v-row>
+
+
+    <div class="section-divider mb-4"></div>
+
+    <h2 class="mb-6">Options</h2>
+
+    <section class="mb-10">
+      <h3 class="mb-3">Resend email</h3>
+
+      <p class="mb-4">
+        We will send another email to the reference. It will include a link to
+        the My ECE Registry to provide a reference for you. It may take a few
+        minutes to receive the email.
+      </p>
+
+      <p class="mb-3">
+        If they do not receive the email:
+      </p>
+
+      <ul class="mb-4 ml-6 bullet-list">
+        <li class="mb-4">Ask the person to check their spam folder</li>
+        <li>Check you provided the correct email address</li>
+      </ul>
+
+      <p class="mb-6">
+        If you need to correct any information, choose to use a new reference
+        below.
+      </p>
+
+      <v-btn
+        variant="outlined"
+        color="primary"
+        :loading="resending"
```

Comment:
this piece you can use the loading store to check if something is being resent. 

here's an example  
:loading="loadingStore.isLoading('draftapplication_put') || loadingStore.isLoading('icra_put')"

https://github.com/bcgov/ECC-ECER/blob/a14af9941bf9bd8e394681790360c5c96d65c6b0/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Declaration.vue#L38